enterprise/utils.py

```diff
@@ -527,3 +532,61 @@ def filter_audit_course_modes(enterprise_customer, course_modes):
         return [course_mode for course_mode in course_modes if course_mode['mode'] not in audit_modes]
     else:
         return course_modes
+
+
+def enterprise_login_required(view):
+    """
+    Decorator for enterprise views.
```

Comment:
Can you make this line a short description of what the decorator does for enterprise views?